core/components/seosuite/src/Processors/Mgr/Redirects/Create.php

```diff
@@ -56,6 +56,7 @@ public function beforeSave()
         }
 
         $this->object->set('new_url', $seosuite->formatUrl($this->getProperty('new_url')));
+        $this->object->set('last_visit', null);
```

Comment:
This is mostly a change to make it work with MySQL 8.*. Default value seems to be invalid for that version. Since the column is nullable, we are forcing it here. Happy to remove and have another MR on that matter if you don't want to mix concerns.